frontends/infinite-corridor/package.json

```diff
@@ -31,7 +31,6 @@
     "ol-util": "workspace:*",
     "ol-widgets": "workspace:*",
     "postcss-loader": "^7.0.1",
-    "ramda": "^0.28.0",
```

Comment:
I also removed all usage of ramda in this PR because:
- it was easy to remove. All of our usages have simple native JS alternatives or analogous lodash functions
- we were including it twice in our bundle accidentally.

If we need it for something, we can definitely add it back.